api/src/pcapi/core/bookings/api.py

```diff
@@ -714,6 +715,12 @@ def mark_as_used_with_uncancelling(booking: Booking, validation_author_type: Boo
     # a rollback if we raise a validation exception.
     # Since I lock the stock, I really want to make sure the lock is
     # removed ASAP.
+    if (
+        booking.deposit
+        and booking.deposit.expirationDate
+        and not booking.deposit.expirationDate >= datetime.datetime.utcnow()
```

Comment:
Simplifions : `not >=` c'est `<` : 
```suggestion
        and booking.deposit.expirationDate < datetime.datetime.utcnow()
```